src/main/kotlin/turku/forge/babyproject/api/RootApiController.kt

```diff
@@ -1,19 +1,69 @@
 package turku.forge.babyproject.api
 
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include
+import org.springframework.hateoas.LinkRelation
+import org.springframework.hateoas.server.core.Relation
+import org.springframework.hateoas.server.mvc.WebMvcLinkBuilder
+import org.springframework.hateoas.server.mvc.linkTo
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.CrossOrigin
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RestController
+import turku.forge.babyproject.config.STOMP_ENDPOINT
+import turku.forge.babyproject.resources.Resource
 
-const val API_PATH = "/"
+const val API_PATH = ""
+
+// @TODO: maybe move these relations to their own ENUM if needed
```

Comment:
I think we should avoid having `TODO`s in our code; it's extra noise and they're often left there indefinitely. They make for perfect issues instead!